### DISCUSSION TITLE
静音下的喇叭右边的空隙太大

### DISCUSSION THREAD

**Discussion by dheart-joe** [linuxdeepin/developer-center#2863] [Features Request & Ideas | 特性请求 & 头脑风暴]
在静音下的喇叭右边的空隙太大，建议借鉴windows的，静音的时候，是喇叭右边有个叉叉，这样不至于，一半是喇叭，一半的空白的空间，看起来很突兀

**Comment by dheart-joe** [comment]
![image](https://user-images.githubusercontent.com/10397085/164968910-dc8e185f-a3c1-42df-bc30-35efba02b99a.png)
无法忍受，静音的图标

**Comment by BLumia** [comment]
从根源上解决的话，需要调整设计，让设计人员提供不存在这种问题的图标（比如右侧用 x 号表示没有声音）。绕过的话，应该先确定这里的图标是否来自所选的图标主题，如果是的话（我记得是的），更改图标主题或者提供图标来覆盖掉这几个图标就行了。

@meiyixiang

**Comment by meiyixiang** [comment]
I have recorded and fed back the problem of icon display and hope to optimize UI in the next version
